component/schedule.libsonnet

```
@@ -41,6 +41,19 @@ local buildSchedule(name, namespace, backupSchedule, pruneSchedule='10 */4 * * *
     secretkeyname: 'password',
   };
 
+  local customCA = if params.customCA != null then {
+    apiVersion: 'v1',
+    kind: 'ConfigMap',
+    metadata: {
+      name: 'k8up-custom-ca',
+      namespace: namespace,
+    },
+    data: {
+      'ca.crt': params.customCA,
+    },
+  };
+  local customCAname = if params.customCA != null then 'k8up-custom-ca' else null;
```

to avoid repeating the same name:

```suggestion
  local customCAname = if params.customCA != null then customCA.metadata.name else null;
```